Add `flag_utils.py`, `pyproject.toml` to configure `pyink`

langmore opened the pull request:
Added `flag_utils.py`
* This is an almost copy of `weatherbench2/flag_utils.py`. I've been using it and it works.
* Useful for defining chunk sizes, and also variable/dim renames, from the command line

Added `pyproject.toml`
* To configure `pyink`

Added a note to `CONTRIBUTING.md`
* Requesting users run `pyink` on their modified files.

raspstephan commented:
Regarding the flag_utils. Right now it seems it's not actually used in any script in WB-X. Do you think you could just use WB2 for this? If we have scripts that use it in WB-X I am happy to add it but I am a little hesitant to just have it floating around in the WB-X codebase without usage. WDYT?

langmore commented:
Good point. I think there is some advantage in not having WBX users depend on WB2. I updated the PR by modifying `run_example_benchmark.py` to use these flags to rename dimensions. That's actually the use case I wanted these utils for.

I also added ".swp" to `.gitignore`
